src/components.ts

```
@@ -217,8 +284,56 @@ export class VaulDrawerContent extends HTMLElement {
+        if (!this.#builtInHandle?.parentNode) return;
+
+        this.#builtInHandle.parentNode.removeChild(this.#builtInHandle);
+        this.#builtInHandle = undefined;
+        logger.debug("VaulDrawerContent: Built-in handle removed");
+    }
+}
+
+export class VaulDrawerHandle extends HTMLElement {
+    constructor() {
+        super();
+    }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)